integration-templates/unanet/nango.yaml

```diff
@@ -0,0 +1,96 @@
+integrations:
+    unanet:
+        actions:
+            create-lead:
+                endpoint: POST /lead
```

Comment:
out of curiosity, do we recommend having action endpoints not prefixed with the integration name?

Reply:
Don't think we have anything official, but whenever you call an action you must pass in the provider config key so prefixing with the integration seems redundant